mypyc/test/test_run.py

```diff
@@ -52,10 +52,17 @@
     'run-bench.test',
     'run-mypy-sim.test',
     'run-dunders.test',
-    'run-singledispatch.test'
+    'run-singledispatch.test',
+    'run-attrs-non-auto.test',
 ]
+if sys.version_info >= (3, 6):
```

Comment:
mypyc doesn't support python before 3.6, we can just merge the two `run-attrs` test file.

Reply:
removed